{
    "source": "BDSA",
    "name": "BDSA-2019-2928",
    "title": "OpenCV Vulnerable to Denial-of-service (DoS) via Out-of-Bounds Read in 'dis_flow.cpp' File",
    "description": "OpenCV contains a denial-of-service (DoS) vulnerability. An attacker could exploit this by supplying a victim with a crafted file that when processed results in a out-of-bounds read that can trigger an application crash.",
    "technicalDescription": "The out-of-bounds read vulnerability occurs in the` cv::hal_baseline::v_load()` function in the `intrin_sse.hpp` file when called from the `cv::computeSSDMeanNorm()` function in `dis_flow.cpp`.",
    "publishedDate": "2019-09-12T12:49:11.136Z",
    "updatedDate": "2019-09-12T12:49:11.125Z",
    "disclosureDate": "2019-09-07T00:00:00.000Z",
    "exploitPublishDate": "2019-09-07T00:00:00.000Z",
    "solution": "",
    "severity": "LOW",
    "cvss2": {
        "baseScore": 4.3,
        "impactSubscore": 2.9,
        "exploitabilitySubscore": 8.6,
        "severity": "LOW",
        "accessVector": "NETWORK",
        "accessComplexity": "MEDIUM",
        "authentication": "NONE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "PARTIAL",
        "temporalMetrics": {
            "exploitability": "PROOF_OF_CONCEPT",
            "remediationLevel": "UNAVAILABLE",
            "reportConfidence": "UNCORROBORATED",
            "score": 3.7
        },
        "vector": "(AV:N/AC:M/Au:N/C:N/I:N/A:P/E:POC/RL:U/RC:UR)"
    },
    "cvss3": {
        "baseScore": 4.3,
        "impactSubscore": 1.4,
        "exploitabilitySubscore": 2.5,
        "severity": "LOW",
        "attackVector": "LOCAL",
        "attackComplexity": "LOW",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "LOW",
        "privilegesRequired": "NONE",
        "scope": "CHANGED",
        "userInteraction": "NONE",
        "temporalMetrics": {
            "exploitability": "PROOF_OF_CONCEPT",
            "remediationLevel": "UNAVAILABLE",
            "reportConfidence": "REASONABLE",
            "score": 3.9
        },
        "vector": "CVSS:3.0/AV:L/AC:L/PR:N/UI:N/S:C/C:N/I:N/A:L/E:P/RL:U/RC:R"
    },
    "useCvss3": true,
    "classifications": [],
    "zeroDay": false,
    "underReview": true,
    "parentAdvisory": false,
    "credit": "RootUp",
    "workaround": "",
    "bdsaTags": [],
    "overallScore": 3.9,
    "_meta": {
        "allow": [
            "GET"
        ],
        "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/BDSA-2019-2928",
        "links": [
            {
                "rel": "cwes",
                "href": "https://trendmicro2.app.blackduck.com/api/cwes/CWE-125"
            },
            {
                "rel": "related-vulnerability",
                "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/CVE-2019-16249",
                "label": "NVD"
            },
            {
                "rel": "bdsa-ranges",
                "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/BDSA-2019-2928/ranges"
            },
            {
                "rel": "reference",
                "href": "https://github.com/opencv/opencv/issues/15481",
                "label": "ADVISORY"
            },
            {
                "rel": "reference",
                "href": "https://github.com/opencv/opencv/issues/15481",
                "label": "POC"
            },
            {
                "rel": "default-remediation-status",
                "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/BDSA-2019-2928/default-remediation-status"
            }
        ]
    }
}